<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoinputDoc</class>
 <widget class="QDialog" name="AutoinputDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArmPetrol</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="2">
    <widget class="WdtContainer" name="wc" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Station</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="C5LineEdit" name="lineEdit_7">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="lvalue" stdset="0">
         <string>station</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Time created</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="C5LineEdit" name="lineEdit_8">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="lvalue" stdset="0">
         <string>autoinput_state</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="C5LineEdit" name="leCode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Date created</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="C5TableWidget" name="tbl">
        <column>
         <property name="text">
          <string>RowId</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fuel id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fuel</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Flash</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ArmPet</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="C5DateEdit" name="leDateCreated">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="C5LineEdit" name="leState">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="lkey" stdset="0">
         <string>autoinput_state</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="leTimeCreated">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leTime">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="C5LineEdit" name="leStation">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="lkey" stdset="0">
         <string>station</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="C5DateEdit" name="leDate">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>State</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>273</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnWrite">
           <property name="text">
            <string>Write</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnWriteDraft">
           <property name="text">
            <string>Write draft</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>272</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>WdtContainer</class>
   <extends>QWidget</extends>
   <header location="global">wdtcontainer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
